docs/grpc.md

```diff
@@ -424,6 +425,21 @@ Response for creating occurrences in batch.
 
 
 
+<a name="rode.v1alpha1.EvaluatePolicyInput"></a>
+
+### EvaluatePolicyInput
```

Comment:
I'm not really sure how I feel about using protobufs to create structs that are only used internally to Rode.  I decided to do this because the JSON we send to OPA contains structs that were already created via protobufs, which forces us to use `protojson.Marshal` instead of `json.Marshal`.  So I felt like creating my own struct that contains occurrences (and later, possibly notes) would only complicate this.

Reply:
> What about keeping this as a normal struct that implements `MarshalJSON`?

I thought about that approach, but it seems like a lot of added complexity for the problem that we're actually trying to solve here.  We'd also have to add to this when notes are eventually added to the policy evaluation input.

> Or maybe have internal protos off in their own package?

This seems like a better approach to me.

Reply:
Created #102